x-pack/test/api_integration/apis/ml/trained_models/get_models.ts

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { USER } from '../../../../functional/services/ml/security_common';
+import { COMMON_REQUEST_HEADERS } from '../../../../functional/services/ml/common_api';
+
+export default ({ getService }: FtrProviderContext) => {
+  const supertest = getService('supertestWithoutAuth');
+  const ml = getService('ml');
+
+  describe('GET trained_models', () => {
+    let testModelIds: string[] = [];
+
+    before(async () => {
+      await ml.testResources.setKibanaTimeZoneToUTC();
+      testModelIds = await ml.api.createdTestTrainedModels('regression', 5, true);
+    });
+
+    after(async () => {
+      await ml.api.cleanMlIndices();
+      // delete created ingest pipelines
+      await Promise.all(testModelIds.map((modelId) => ml.api.deleteIngestPipeline(modelId)));
+    });
+
+    it('returns all trained models with associated pipelined', async () => {
```

Comment:
Typo - should just be `pipelines`

Reply:
fixed in https://github.com/elastic/kibana/pull/104819/commits/e13787bb4dfb2224d27f44d90ad1150b971d8fee